src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -409,37 +409,40 @@ void Select_PrintMenu(SelectContext* this, GfxPrint* printer) {
     GfxPrint_Printf(printer, "OPT=%d", this->opt);
 }
 
-static char* sLoadingMessages[] = {
-    "\x8Dｼﾊﾞﾗｸｵﾏﾁｸﾀﾞｻｲ", // "Please wait a minute"
-    "\x8Dﾁｮｯﾄ ﾏｯﾃﾈ",     // "Hold on a sec"
-    "\x8Cｳｪｲﾄ ｱ ﾓｰﾒﾝﾄ",  // "Wait a moment"
-    "\x8Cﾛｰﾄﾞ\x8Dﾁｭｳ",   // "Loading"
-    "\x8Dﾅｳ ﾜｰｷﾝｸﾞ",     // "Now working"
-    "\x8Dｲﾏ ﾂｸｯﾃﾏｽ",     // "Now creating"
-    "\x8Dｺｼｮｳｼﾞｬﾅｲﾖ",    // "It's not broken"
-    "\x8Cｺｰﾋｰ ﾌﾞﾚｲｸ",    // "Coffee Break"
-    "\x8C"
-    "Bﾒﾝｦｾｯﾄｼﾃｸﾀﾞｻｲ",                      // "Please set B side"
-    "\x8Dｼﾞｯﾄ\x8Cｶﾞﾏﾝ\x8Dﾉ\x8Cｺ\x8Dﾃﾞｱｯﾀ", // "Be patient, now"
-    "\x8Dｲﾏｼﾊﾞﾗｸｵﾏﾁｸﾀﾞｻｲ",                 // "Please wait just a minute"
-    "\x8Dｱﾜﾃﾅｲｱﾜﾃﾅｲ｡ﾋﾄﾔｽﾐﾋﾄﾔｽﾐ｡",          // "Don't worry, don't worry. Take a break, take a break"
+// clang-format off
+static const char* sLoadingMessages[] = {
+    "\x8Dｼﾊﾞﾗｸｵﾏﾁｸﾀﾞｻｲ", // "Please wait a minute",
+    "\x8Dﾁｮｯﾄ ﾏｯﾃﾈ", // "Hold on a sec",
+    "\x8Cｳｪｲﾄ ｱ ﾓｰﾒﾝﾄ", // "Wait a moment",
+    "\x8Cﾛｰﾄﾞ\x8Dﾁｭｳ", // "Loading",
+    "\x8Dﾅｳ ﾜｰｷﾝｸﾞ", // "Now working",
+    "\x8Dｲﾏ ﾂｸｯﾃﾏｽ", // "Now creating",
+    "\x8Dｺｼｮｳｼﾞｬﾅｲﾖ", // "It's not broken",
+    "\x8Cｺｰﾋｰ ﾌﾞﾚｲｸ", // "Coffee Break",
+    "\x8C" "Bﾒﾝｦｾｯﾄｼﾃｸﾀﾞｻｲ", // "Please set B side",
+    "\x8Dｼﾞｯﾄ\x8Cｶﾞﾏﾝ\x8Dﾉ\x8Cｺ\x8Dﾃﾞｱｯﾀ", // "Be patient, now",
+    "\x8Dｲﾏｼﾊﾞﾗｸｵﾏﾁｸﾀﾞｻｲ", // "Please wait just a minute",
+    "\x8Dｱﾜﾃﾅｲｱﾜﾃﾅｲ｡ﾋﾄﾔｽﾐﾋﾄﾔｽﾐ｡", // "Don't worry, don't worry. Take a break, take a break",
 };
+// clang-format on
 
 void Select_PrintLoadingMessage(SelectContext* this, GfxPrint* printer) {
     s32 randomMsg;
 
-    GfxPrint_SetPos(printer, 0xA, 0xF);
+    GfxPrint_SetPos(printer, 10, 15);
     GfxPrint_SetColor(printer, 255, 255, 255, 255);
     randomMsg = Rand_ZeroOne() * ARRAY_COUNT(sLoadingMessages);
     GfxPrint_Printf(printer, "%s", sLoadingMessages[randomMsg]);
 }
 
-static char* sAgeLabels[] = {
-    "\x8D"
-    "17(ﾜｶﾓﾉ)", // "17(young)"
-    "\x8D"
-    "5(ﾜｶｽｷﾞ)", // "5(very young)"
+// clang-format off
+static const char* sAgeLabels[] = {
+    // "17(young)"
+    "\x8D" "17(ﾜｶﾓﾉ)",
+    // "5(very young)"
+    "\x8D" "5(ﾜｶｽｷﾞ)",
 };
+// clang-format on
```

Comment:
i would move these to the side as well (and remove clang comments)

Reply:
Removing the clang-format comments does this. Is it fine?
```c

static const char* sAgeLabels[] = {
    "\x8D"
    "17(ﾜｶﾓﾉ)", // "17(young)"
    "\x8D"
    "5(ﾜｶｽｷﾞ)", // "5(very young)"
};

```

Reply:
i would say so. in general we try to avoid disabling clang unless we really need to.
we may experiment with other formatters or format rules in the future which will fix tihs problem (and the many other issues some people have with it)
but for now this isnt too much of a big deal imo